modules/Sources/Dependencies/WalletStorage/WalletStorage.swift

```diff
@@ -206,7 +226,7 @@ public struct WalletStorage {
     @discardableResult
     public func deleteData(
         forKey: String,
-        account: String = ""
+        account: String
```

Comment:
After we change the Queries and Uniqueness Constraints, does that leave sensitive data around under the old keys even if the user nukes the wallet?